From 373cf3ddfeb569a1fff99bf7ce52870ad81e44a7 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Wed, 24 Oct 2018 16:34:26 +0300
Subject: [PATCH 0734/1051] ARM64: dts: marvell: Enable USB 2.0 ports on A3900
 DB

Enable USB 2.0 ports on Armada-3900-DB board setups A and B.
Remove reference to non-existing PHY connection from second USB port.

Change-Id: I4b92e80bc6edd5c55ca7e2b1054f68d2c0083fcd
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/60544
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Igal Liberman <igall@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-3900-db-vd-A.dts | 2 +-
 arch/arm64/boot/dts/marvell/armada-3900-db-vd-B.dts | 2 +-
 arch/arm64/boot/dts/marvell/armada-3900-db.dtsi     | 2 --
 3 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-3900-db-vd-A.dts b/arch/arm64/boot/dts/marvell/armada-3900-db-vd-A.dts
index 541bde9682c3..28398eb96ca2 100644
--- a/arch/arm64/boot/dts/marvell/armada-3900-db-vd-A.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3900-db-vd-A.dts
@@ -95,7 +95,7 @@
 };
 
 &cp0_usb3_1 {
-	status = "disabled";
+	status = "okay";
 };
 
 &cp0_pcie0 {
diff --git a/arch/arm64/boot/dts/marvell/armada-3900-db-vd-B.dts b/arch/arm64/boot/dts/marvell/armada-3900-db-vd-B.dts
index f1e42fdffa6b..06a08275fee8 100644
--- a/arch/arm64/boot/dts/marvell/armada-3900-db-vd-B.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3900-db-vd-B.dts
@@ -96,7 +96,7 @@
 };
 
 &cp0_usb3_1 {
-	status = "disabled";
+	status = "okay";
 };
 
 &cp0_pcie0 {
diff --git a/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi b/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
index 4870efa18504..dde5993de18e 100644
--- a/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
@@ -138,8 +138,6 @@
 &cp0_usb3_1 {
 	status = "disabled";
 	usb-phy = <&cp0_usb3_1_phy>;
-	phys = <&cp0_comphy4 1>;
-	phy-names = "usb";
 };
 
 &cp0_pcie0 {
-- 
2.17.1

